Services/Repositories/ConversationRepository.cs

```diff
@@ -49,6 +55,31 @@ public async Task CloseConversationAsync(Conversation conversation)
             await _dbContext.SaveChangesAsync();
         }
 
+        public async Task CloseUnusedConversationsAsync()
+        {
+
+            var inProgressThresholdMin = _configuration.GetValue<int>("Hangfire:closeInProgressConversationThresholdMin");
+            var inNewThresholdMin = _configuration.GetValue<int>("Hangfire:closeNewConversationThresholdMin");
+
+            var inProgressThresholdDate = DateTime.UtcNow.AddMinutes(-inProgressThresholdMin);
+            var newThresholdDate = DateTime.UtcNow.AddMinutes(-inNewThresholdMin);
+
+            var conversationsToClose = await _dbContext.Conversations
+                                        .Where(c => (c.Status == ConversationStatus.IN_PROGRESS &&
+                                                      c.LastMessage.CreateDate < inProgressThresholdDate)
+                                                      ||
+                                                      (c.Status == ConversationStatus.NEW &&
+                                                       c.LastMessage.CreateDate < newThresholdDate)
+                                                ).ToListAsync();
+
+            for (var i = 0; i < conversationsToClose.Count; i++)
```

Comment:
`foreach(var conversation in conversationsToClose) 
{
    conversation.Status = ConversationStatus.DONE;
}`